src/auth/project-config.ts

```diff
@@ -57,13 +66,19 @@ export class ProjectConfig {
    * This is based on the calling code of the destination phone number.
    */
   public readonly smsRegionConfig?: SmsRegionConfig;
+  /**
+   * The Multi Factor Config for the project.
+   * Configures the multi factor settings for the project.
+   * Supports only phone and TOTP
```

Comment:
nit: missing "." at the end.

Also, let's add a comment here - "This maps to the "mfa" field in the backend request."

Reply:
This doesn't reflect in the diff yet.